src/core/cluster.py

```diff
@@ -395,6 +401,20 @@ def bootstrap_server(self) -> str:
             )
         )
 
+    @property
+    def controller_quorum_uris(self) -> str:
+        """The current controller quorum uris when running KRaft mode."""
+        # FIXME: when running controller node.id will be unit.id + 100. If unit is only running
+        # the broker node.id == unit.id. This way we can keep a human readable mapping of ids.
+        if self.kraft_mode and self.runs_controller:
```

Comment:
**question**: Do we need both checks here? Considering the comment above I would expect `self.runs_controller` to be necessary and sufficient